code/game/machinery/kitchen/microwave.dm

```diff
@@ -27,114 +38,111 @@
 /obj/machinery/microwave/Initialize()
 	. = ..()
 	create_reagents(100)
+	soundloop = new(list(src), FALSE)
```

Comment:
Not seeing where this reference is cleared in `Destroy`.

Reply:
d'oh

Reply:
```suggestion
	soundloop = new(list(src), FALSE)

/obj/machinery/microwave/Destroy()
	dispose()
	QDEL_NULL(soundloop)
	..()
```
This look good to you?